input/chainsync/options.go

```diff
@@ -101,13 +101,6 @@ func WithStatusUpdateFunc(
 	}
 }
 
-// WithBulkMode specifies whether to use the "bulk" sync mode with NtN (node-to-node). This should only be used against your own nodes for resource usage reasons
-func WithBulkMode(bulkMode bool) ChainSyncOptionFunc {
```

Comment:
Instead of removing this, I think we should emit a warning log... at least for now.